docs-src/clusters/how-to-integrate-elasticsearch-into-a-temporal-cluster.md

```diff
@@ -8,19 +8,25 @@ tags:
   - filtered-lists
   - visibility
 ssdi:
-- Elasticsearch v8 is supported beginning with Temporal Server version 1.18.0
-- Elasticsearch v7.10 is supported beginning with Temporal Server version 1.17.0
-- Elasticsearch v6.8 is supported through Temporal Server version 1.17.x
-- Elasticsearch v6.8 and v7.10 are explicitly supported with AWS Elasticsearch
+  - Elasticsearch v8 is supported beginning with Temporal Server version 1.18.0
+  - Elasticsearch v7.10 is supported beginning with Temporal Server version 1.17.0
+  - Elasticsearch v6.8 is supported through Temporal Server version 1.17.x
+  - Elasticsearch v6.8 and v7.10 are explicitly supported with AWS Elasticsearch
 ---
 
-[Advanced Visibility](/concepts/what-is-advanced-visibility) features depend on an integration with Elasticsearch.
+You can integrate Elasticsearch with your Temporal Cluster for [Advanced Visibility](/concepts/what-is-advanced-visibility) to take on the Visibility request load. Elasticsearch is recommended for large-scale operations on the Temporal Cluster, and is required if you use Cassandra as your [Persistence store](/concepts/what-is-a-temporal-cluster#persistence).
+
+Note that if you use MySQL or PostgreSQL as your Visibility store, Temporal Server version 1.20 and later supports Advanced Visibility features on MySQL (version 8.0.17 and later) and PostgreSQL (version 12 and later).
+To enable Advanced Visibility on your SQL databases, ensure that you do the following:
+
+- Upgrade your Temporal Server to version 1.20.
+- Update your MySQL to version 8.0.17 (or later) or PostgreSQL version 12 (or later).
```

Comment:
I think we should also point out the specific `pluginName` setting here.
For example, if they set `mysql` instead of `mysql8`, it won't work with advanced visibility features. Same with `postgres` vs `postgres12`.

Reply:
yeah; I think I'll call that out.